rhtconsulting/rhc-ose
# rhc-htpasswd role

#### What does this PR do?

This PR is used to create specified demo users in the designated htpasswd file.
#### How should this be manually tested?

simply head to `playbooks/rhc-htpasswd`, add your test host under host in the `test/inv` file and run the following command:
`ansible-playbook -i test/inv htpasswd.yaml` 

Then verify that logins of the users `joe` and `alice` are added to the htpasswd file(default /etc/origin/openshift-passwd) using: `htpasswd -v <htpasswdFile> <user>`
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @sabre1041  @oybed
